tempus_fugit_minecraft/player.py

```diff
@@ -1,4 +1,4 @@
-from tempus_fugit_minecraft.utilities import WHOLE_WORLD_SIZE, WORLD_SIZE
+from tempus_fugit_minecraft.utilities import WORLD_SIZE
 from tempus_fugit_minecraft.block import BRICK, GRASS, SAND, TREE_TRUNK, TREE_LEAVES
```

Comment:
you can add create light clouds and dark clouds on keys 4,5 and keep Tree and leave for 6 and 7 because you create clouds first and it was ready to add in this line.